timescale-docker/README.md

```diff
@@ -0,0 +1,38 @@
+# Restore database from backups
+
+Below is instructions how to spin up CCD scan database locally which is restored from Stage- or Testnet backups.
+
+Before following below instructions make sure you have read access to the S3 buckets where the backups are located.
+
+## Database restore
+
+Build custom docker image by running this
+```
+cd ./docker
+docker build -t timescale-restore:1 .
+cd ..
+```
+
+Make a `pgbackrest.conf` file like `pgbackrest.conf.example` replacing `$AWS_ACCESS_KEY` and `$AWS_SECRET_KEY` with your access credentials.
+
+Start docker-compose
+```
+docker-compose up
+```
+
+Exec into the container.
+```
+docker container ls | grep timescale | awk '{print $1}' | head -n 1 | xargs -o -I {} docker exec -it {} bash
```

Comment:
This will work some of the time on some machines.

I recommend not having needlessly complex commands in READMEs, commands which don't really work reliably.

Use

```
docker-compose exec timescaledb bash
```

instead.